kyo-core/shared/src/main/scala/kyo/Clock.scala

```diff
@@ -104,21 +118,163 @@ object Clock:
     def let[A, S](c: Clock)(f: => A < S)(using Frame): A < S =
         local.let(c)(f)
 
+    /** Gets the current Clock instance from the local context.
+      *
+      * This is the primary way to access the current Clock instance when you only need to use it directly.
+      *
+      * @return
+      *   The current Clock instance
+      */
+    def get(using Frame): Clock < Any =
+        use(identity)
```

Comment:
Any reason to avoid local.get?
```suggestion
        local.get
```